### ISSUE TITLE
More than just time and heartrate

### ISSUE THREAD

**Issue by paulmach** [mapbox/geojson-coordinate-properties#3]
At strava, for each track point we could have data on time, distance, elevation, heartrate, cadence (running or cycling), power, temperature, grade and speed. Note that distance and speed could be those collected from sensors and different than that computed from the latlng and time. So I would suggest including requirements for the names of those extra attributes. 

Being all inclusive would be hard since FIT, for example, has about 40 extra attributes you can assign to a trackpoint that can get pretty crazy like total_hemoglobin_conc, left_torque_effectiveness, etc. The full list is in the fit sdk, I can send it if you want http://www.thisisant.com/resources/fit  

I know GPX doesn't include all these attributes officially, but people have started to include them in different ways. What we support at Strava can be found here. http://strava.github.io/api/v3/uploads/#gpx---gps-exchange-format

Basically, it would be good to agree on the names so Strava doesn't call it `temp` while runkeeper calls it `temperature`.

**Comment by tmcw** [comment]
:+1: seems like expanding the set of known properties to a "90% of what you can represent as sports data" makes sense.

**Comment by jmoe** [comment]
@paulmach beat me to it, but also said it much better than I would have. +1
also, short of going full-on schema, it would also be useful to agree upon acceptable values for the extra attributes.

**Comment by bhousel** [comment]
Just found this project today.  I'd like to store metadata about large image repositories in GeoJSON, and would use `coordinateProperties` to tag things related to the images that were taken (e.g. the more useful exif tags:  image dimensions, capture time, gpstrack/bearing/angle, gpsdop, accelerometer data).

Maybe I'm looking in the wrong place, but at least for my use it would be fantastic if this project weren't limited to tracking fitness data.

**Comment by halset** [comment]
Perhaps not limiting possible coordinate attribute names? Some might want to store turn radius on a vessel route or any other thing that we have not thought of?

**Comment by tmcw** [comment]
To clarify, this isn't limited to any set of attributes. From the spec:

> Members of the coordinateProperties property can have any name, but semantics of two names are defined:

The objective is that you can use any properties with the coordinate properties style, but a few of those properties (times & heart) have agreed-upon semantics, and it's encouraged to use `heart` for heart rate data, rather than using the `heart` property for something else.

But you can use any property names and any other meanings for any other data.

**Comment by halset** [comment]
@tmcw, thanks! And sorry for the confusion.
